server/svgs.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {

```

Comment:
Nice, maybe I can also fetch from this for my service?

Reply:
@phucciTr once it's all up and running, absolutely!